text/logs/0114-log-correlation.md

```diff
@@ -0,0 +1,137 @@
+| traceid    | Yes      | 16-byte numeric value as Base16-encoded string
+| spanid     | Yes      | 8-byte value represented as Base16-encoded string
+| traceflags | No       | 8-bit numeric value as a Base16-encoded string
+
+### Resource Correlation
+As important as what was occurring in a program’s execution is, where it was 
+occurring is just as important. Resource correlation allows a log entry to be 
+associated with an infrastructure resource, and in turn system and program 
+metrics that describe the wider program state. The form that the resource takes 
+is also more diverse than the tracing scope- the resource may be a pod running 
+in a Kubernetes cluster, a virtual machine running in a cloud, a serverless 
+lambda, or an old-school server sitting in a data center. An application 
+environment may be an orchestration of multiple types of resource working 
+together.
+
+From a logging standpoint, a resource is also almost always a constant within an 
+application process- a container may not have the same identifier on every run, 
+but it does keep that identifier while it’s running, and that resource 
+identifier is constant for every log entry created on that resource. As a 
+result, resource correlation may not happen at the log entry level, so we may or 
+may not put the resource correlation information in the log entry itself.
+
+Resource information may be managed as part of the log ingestion process- for 
+example a Docker logging driver will know which container logs came from. 
+Full resource information may also not be available, as when logs are 
+aggregated by syslog or a similar system that has less context available to it. 
+As a result resource correlation information in the logs entries themselves 
+should be considered optional. If included, the resource information should 
+follow the semantic conventions for resources.
+
+If the log entry is expressed in key-value pairs, any resource keys should be 
+prepended with ‘resource’- for example, `resource.service.name=”shoppingcart”`. 
+If the entry is expressed in JSON, the resource key-values should be placed in 
+an object named “resource” at the top level of the object.
+
+### Correlation Context
+
+A Correlation Context is a set of key-value pairs that is shared amongst the 
+spans of a distributed trace. Like span attributes, the correlation context may 
+already be carried with span information, so duplicating this information may be 
+redundant. In certain cases it may be important to associate this context with 
+log entries. When the context is embedded in a log entry, the key-value
+pairs should be placed in a ‘correlation’ namespace. Where key-value pairs are 
+supported, embed the correlation key as “correlation.key_name”. In JSON or 
+other formats that allow nested structures, the key-value pairs should be 
+placed in an object named ‘correlation’ at the top of the object.
+
+
+## Internal details
+
+### Encoding
```

Comment:
The content of this section are 2 examples. Perhaps use "Examples" as the section title instead of "Encoding"?